core/common/src/main/java/alluxio/security/user/BaseUserState.java

```diff
@@ -67,6 +67,15 @@ public Subject getSubject() {
     return mSubject;
   }
 
+  /**
+   * Attempts to get the user.
```

Comment:
"Attempts to get the user for this instance."

Reply:
Fixed.